client/state/data-layer/wpcom/me/account/close/test/index.js

```diff
@@ -11,8 +11,8 @@ import { http } from 'calypso/state/data-layer/wpcom-http/actions';
 import { closeAccount } from 'calypso/state/account/actions';
 import { ACCOUNT_CLOSE_SUCCESS } from 'calypso/state/action-types';
 
-jest.mock( 'calypso/lib/user', () => () => {
-	return { clear: jest.fn() };
+jest.mock( 'calypso/lib/user/store', () => {
+	return { clearStore: jest.fn() };
 } );
```

Comment:
Can this mock be removed, too? Now the data layer handler or any other Redux code doesn't call `clearStore()`. Only the React UI does.

Reply:
Of course. Thanks for noticing. Cleaned up in 503a226ef77a7fdac03b2290e3bdb22060b9f725